Quaver.Shared/Screens/Edit/Input/EditorInputConfig.cs

```diff
@@ -0,0 +1,184 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.Xna.Framework.Input;
+using Quaver.Shared.Config;
+using RestSharp;
+using Wobble.Logging;
+using Wobble.Platform;
+using YamlDotNet.Core;
+using YamlDotNet.Serialization;
+
+namespace Quaver.Shared.Screens.Edit.Input
+{
+    [Serializable]
+    public class EditorInputConfig
+    {
+        [YamlIgnore] public static string ConfigPath = ConfigManager.GameDirectory.Value + "/editor_keys.yaml";
+
+        public bool ReverseScrollSeekDirection { get; private set; }
+        public Dictionary<KeybindActions, KeybindList> Keybinds { get; private set; }
+        public Dictionary<string, KeybindList> PluginKeybinds { get; private set; }
+
+        public EditorInputConfig()
+        {
+            ReverseScrollSeekDirection = true;
+            Keybinds = DefaultKeybinds;
+            PluginKeybinds = new Dictionary<string, KeybindList>();
+        }
+
+        public static EditorInputConfig LoadFromConfig()
+        {
+            var config = new EditorInputConfig();
+
+            if (!File.Exists(ConfigPath))
+            {
+                Logger.Debug("No editor key config found, using default", LogType.Runtime);
+                config.SaveToConfig();
+            }
+            else
+            {
+                try
+                {
+                    using (var file = File.OpenText(ConfigPath))
+                    {
+                        config = Deserialize(file);
+                    }
+
+                    Logger.Debug("Loaded editor key config", LogType.Runtime);
+                    config.SaveToConfig(); // Reformat after loading
+                }
+                catch (Exception e)
+                {
+                    Logger.Error($"Could not load editor key config, using default: {e.Message}", LogType.Runtime);
+                }
+            }
+
+            return config;
```

Comment:
```suggestion
            if (!File.Exists(ConfigPath))
            {
                Logger.Debug("No editor key config found, using default", LogType.Runtime);
                config.SaveToConfig();
                return config;
            }

            try
            {
                using (var file = File.OpenText(ConfigPath))
                    config = Deserialize(file);

                Logger.Debug("Loaded editor key config", LogType.Runtime);
                config.SaveToConfig(); // Reformat after loading
            }
            catch (Exception e)
            {
                Logger.Error($"Could not load editor key config, using default: {e.Message}", LogType.Runtime);
            }

            return config;
```

Just to reduce nesting a bit.